include/inviwo/core/io/serialization/deserializer.h

```diff
@@ -52,7 +52,7 @@ class FactoryBase;
 template <typename T, typename K>
 class ContainerWrapper;
 
-class IVW_CORE_API Deserializer : public SerializeBase {
+class IVW_CORE_API Deserializer : public SerializeBase, public Logger {
```

Comment:
Derive from LogFilter, then you dont need to add any impl here...

Reply:
good point!